src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,14 @@
+.stars {
+  display: flex;
+  gap: 4px;
+
+  &__star  {
+    width: 16px;
+    height: 16px;
+    background-image: url(../images/star.svg) no-repeat center;
+
+    &:nth-child(-n+4)  {
```

Comment:
What about other modifiers? It should work the same with different amounts of active stars